openapi.yml

```diff
@@ -62,6 +62,68 @@ paths:
           description: FlowMod messages sent.
         '400':
            description: Invalid JSON flow.
+  '/api/kytos/flow_manager/v2/flows_by_switch':
+    post:
+      tags:
+        - Add
+      summary: Insert flows by switch.
+      parameters:
+        - name: force
+          schema:
+            type: string
+          description: Option to ignore switch connection errors.
+          required: false
+          example: "true"
+      requestBody:
+        description: Flows by the especified switch.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                switch_id:
+                  $ref: '#/components/schemas/FlowsBody'
+      responses:
+        '202':
+          description: FlowMod messages sent.
+        '400':
+           description: Invalid JSON flow.
+        '404':
+          description: Datapath not found.
+        '424':
+          description: Switch connection error.
+    delete:
+      tags:
+        - Delete
+      summary: Remove flows by matched switch.
+      parameters:
+        - name: force
+          schema:
+            type: string
```

Comment:
Same as the force `boolean` type comment.

Reply:
Changed